### PR TITLE
translated index.md to german

### PR DESCRIPTION
**Pull request by luebke-dev**

### CODE REVIEW

**Comment by Kludex** [comment]
@SebastianLuebke You might want to check [this issue](https://github.com/tiangolo/fastapi/issues/1497) or create a new one talking about the possibility of supporting German. If there's already an issue with it, ignore me :)

File: docs/de/docs/index.md [hunk 1]
```diff
@@ -0,0 +1,442 @@
...
+
+---
+
+## **Typer**, das FastAPI der CLIs
+
+<a href="https://typer.tiangolo.com" target="_blank"><img src="https://typer.tiangolo.com/img/logo-margin/logo-margin-vector.svg" style="width: 20%;"></a>
+
+Wenn Sie eine <abbr title="Kommandozeileninterface">CLI</abbr> Anwendung erstellen, probieren Sie <a href="https://typer.tiangolo.com/" class="external-link" target="_blank">**Typer**</a> aus.
+
+**Typer** ist soetwas wie FastAPI's kleiner Bruder und soll das **FastAPI der CLIs** sein. ⌨️ 🚀
```
Comment (JeromeK13) [inline docs/de/docs/index.md:91]: "soetwas" -> so etwas

File: docs/de/docs/index.md [hunk 2]
```diff
@@ -0,0 +1,442 @@
...
+
+### Alternative API Dokumentation
+
+Wenn Sie jetzt die URL <a href="http://127.0.0.1:8000/redoc" class="external-link" target="_blank">http://127.0.0.1:8000/redoc</a> aufrufen, sehen Sie eine alternative Dokumentaion (bereitgestellt durch <a href="https://github.com/Rebilly/ReDoc" class="external-link" target="_blank">ReDoc</a>).
+
+![ReDoc](https://fastapi.tiangolo.com/img/index/index-02-redoc-simple.png)
+
+## Bespiel Aktualisierung
+
+Verändern Sie nun die Datei `main.py` um den Body von einer `PUT` Anfrage zu empfangen.
```
Comment (JeromeK13) [inline docs/de/docs/index.md:238]: "Verändern Sie nun die Datei `main.py` um den Body von einer `PUT` Anfrage zu empfangen." ->

Verändern Sie nun die Datei `main.py` um den Body einer `PUT` Anfrage zu empfangen

**Review by JeromeK13 — COMMENTED** [review]
Some little typos I've found.

File: docs/de/docs/index.md [hunk 3]
```diff
@@ -0,0 +1,442 @@
...
+Verwendet von FastAPI / Starlette:
+
+- <a href="http://www.uvicorn.org" target="_blank"><code>uvicorn</code></a> - der Server der Ihr Programm lädt und bereitstellt.
+- <a href="https://github.com/ijl/orjson" target="_blank"><code>orjson</code></a> - wird benötigt falls `ORJSONResponse` verwendet werden soll..
+
+Sie können alle diese Pakete mit dem Befehl `pip install fastapi[all]` installieren.
+
+## Lizenz
+
+Dieses Projekt ist unter den Bedingungen der MIT Lizenz lizensiert.
```
Comment (kuchosauronad0) [inline docs/de/docs/index.md:442]: lizensiert -> lizenziert

**Review by kuchosauronad0 — CHANGES_REQUESTED** [review]
I've found a few typos. The rest looks good I guess :) Thanks for putting the work into the translation

**Comment by SebastianLuebke** [comment]
So many typos :D will fix them today

**Comment by SebastianLuebke** [comment]
Fixed the typos now

**Pull request closed without merging**